test/integration/targets/special_vars/meta/main.yml

```diff
@@ -1,2 +1,3 @@
 dependencies:
   - prepare_tests
+  - include_parent_role_vars
```

Comment:
Is this needed for the test? It looks like it's just here to indicate the dependency is required. If that's the case, add an entry to the `aliases` file instead: `needs/target/include_parent_role_vars`